### PR TITLE
Create calcy2.sh

### PR DESCRIPTION
**Pull request by Bukter**
creating new file on ur branch

### CODE REVIEW

File: calcy2.sh [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+1)sum=`expr $n1 + $n2`
+   echo "Sum ="$sum;;
+
+2)sum=`expr $n1 - $n2`
+   echo "Sub = "$sum;;
+
+3)sum=`expr $n1 \* $n2`
+   echo "Mul = "$sum;;
+
+4)sum=`expr $n1 - $n2`
```
Comment (appars) [inline calcy2.sh]: @Bukter Can you please add / instead of .?

**Review by appars — CHANGES_REQUESTED** [review]
Please check my review comments.

File: calcy2.sh (hunk 1 above)
Comment (Bukter) [reply calcy2.sh]: chang r don

**Review by appars — APPROVED** [review]
LGTM

**Pull request merged**